packages/utils/src/normalize.ts

```diff
@@ -72,16 +72,16 @@ export function normalizeToSize<T>(
 function visit(
   key: string,
   value: unknown,
-  depth: number = +Infinity,
-  maxProperties: number = +Infinity,
+  depth = +Infinity,
+  maxProperties = +Infinity,
   memo: MemoFunc = memoBuilder(),
 ): Primitive | ObjOrArray<unknown> {
   const [memoize, unmemoize] = memo;
 
   // Get the simple cases out of the way first
   if (
     value == null || // this matches null and undefined -> eqeq not eqeqeq
-    (['number', 'boolean', 'string'].includes(typeof value) && !isNaN(value))
+    (['number', 'boolean', 'string'].includes(typeof value) && !Number.isNaN(value))
```

Comment:
OK, I wont repeat this anymore, but for everything that's not node only we can't use `Number.isNaN` 😅